client/network_connect_test.go

```diff
@@ -87,6 +87,10 @@ func TestNetworkConnect(t *testing.T) {
 				return nil, fmt.Errorf("expected 'container_id', got %s", connect.Container)
 			}
 
+			if connect.EndpointConfig == nil {
```

Comment:
This checking is virtually covered in `if connect.EndpointConfig.NetworkID != "NetworkID"`